src/Helpers/ClassAliasAutoloader.php

```diff
@@ -0,0 +1,152 @@
+<?php
```

Comment:
I wouldn't duplicate from Tinker. if the project is Laravel then it has Tinker in it and you can just check if the Class exists and use it instead of copying it